cluster/src/main/java/com/hartwig/pipeline/tools/ToolInfo.java

```diff
@@ -0,0 +1,64 @@
+package com.hartwig.pipeline.tools;
+
+import static java.lang.String.format;
+
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+
+public enum ToolInfo {
+
+    AMBER("3.9", 32, 64, 16, false),
+    BAM_TOOLS("1.1.rc1", 16, 24, 16, false), // 1.0 -> 1.1
+    CHORD("2.02_1.14"),
+    COBALT("1.13", 16, 24, 16, false),
+    CUPPA("1.8"),
+    GRIDSS("2.13.2"),
+    GRIPSS("2.3.5", 16, 24, 1, false), // 2.3.5 -> 2.4
+    HEALTH_CHECKER("3.4"),
+    LILAC("1.5.rc1", 16, 24, 8, false), // 1.4.2 -> 1.5
+    LINX("1.24.rc1", 8, 16, 1, false), // 1.23.6 -> 1.24
+    MARK_DUPS("1.0", 16, 24, 16, true), // initial version
+    ORANGE("2.5.2", 16, 18, 4, true), // 2.5.0 -> 2.5.2
+    PAVE("1.5.rc1", 16, 24, 1, false), // 1.4.5 -> 1.5
+    PEACH("1.7"),
+    PROTECT("2.3.1"),
+    PURPLE("3.9", 24, 32, 8, true), // 3.8.4 -> 3.9
+    ROSE("1.3.1", 8, 10, 2, false),
+    SAGE("3.3.rc1   ", 60, 64, 16, false), // 3.2.5 -> 3.3
+    SIGS("1.1"),
+    SV_PREP("1.2.rc1", 48, 64, 24, false), // 1.1 -> 1.2, upgrade with gridss script
+    VIRUSBREAKEND_GRIDSS("2.13.2"),
+    VIRUS_INTERPRETER("1.2");
+
+    public static final int DEFAULT_MAX_HEAP = 4;
+    public static final int DEFAULT_MEMORY = 8;
+
+    public static final String PILOT_VERSION = "pilot"; // will pick up the jar from /opt/toolName/pilot/toolName.jar
+
+    public final String ToolName;
```

Comment:
I don't think this is a question of code style - making const member variables public saves writing an accessor.

Reply:
The capitalization is style but the private-ness is a standard. The argument is that exposing member variables breaks the encapsulation of the object by exposing how data is stored in the object. An object exposes state or functionality always via its public methods and not directly via its variables. 

I agree the value-add of this can be debated for value objects (especially when fields are final), but following this rule uniformly across a code-base in my experience improves the modularity and over maintainability. I would appreciate that we don't introduce any exceptions to this rule, you can generate your accessors using intellij or use an @Immutable.